common/src/stack/report-system/command/report/system/__init__.py

```diff
@@ -58,17 +62,17 @@ def run(self, params, args):
 		tests = glob('tests/*')
 
 		# make it real ugly.
-		if exitonfail == True and pretty == False:
-			_return_code = main(['-v', '-x', *tests])
+		if exitonfail and not pretty:
+			_return_code = main(['-v', '-x', *args, *tests])
```

Comment:
We should use longopts where available when calling out other commands.

Reply:
Not sure what you mean by "longopts". Would you please elaborate?

Reply:
--verbose instead of -v.  That one's not as big of a deal, because it tends to be common, but I've no idea what -s does without hitting the man page for pytest.  In general, when we shell out (which this is sort of doing) we should use the longopts, since it's more clear what the code is doing.

Reply:
Ok I can change that in another PR, but that was already existing so out of scope of this PR.

Reply:
I went ahead and changed this because changing our development process is not a sword I want to die on today.